src/app/app.module.ts

```diff
@@ -69,6 +76,8 @@ export const ROUTES: Routes = [{
         ProfileService,
         SidenavComponent,
         TitleService,
+        HttpService,
```

Comment:
Please do `[{ provide: Http, useClass: HttpService}]` instead. So we can inject the ususal Http